noir/noir-repo/tooling/profiler/src/flamegraph.rs

```diff
@@ -271,30 +280,36 @@ mod tests {
             BTreeMap::default(),
         );
 
-        let samples_per_opcode = vec![10, 20, 30];
-
-        let expected_folded_sorted_lines = vec![
-            "main.nr:2:9::fn main();main.nr:3:13::foo();main.nr:8:13::baz();main.nr:14:13::whatever();opcode::arithmetic 10".to_string(),
-            "main.nr:2:9::fn main();main.nr:4:13::bar();main.nr:11:13::whatever();opcode::arithmetic 20".to_string(),
-            "main.nr:2:9::fn main();main.nr:5:13::whatever();opcode::memory::init 30".to_string(),
+        let samples: Vec<Sample<FieldElement>> = vec![
+            Sample {
+                opcode: AcirOrBrilligOpcode::Acir(AcirOpcode::AssertZero(Expression::default())),
+                call_stack: vec![OpcodeLocation::Acir(0)],
+                count: 10,
+            },
+            Sample {
+                opcode: AcirOrBrilligOpcode::Acir(AcirOpcode::AssertZero(Expression::default())),
+                call_stack: vec![OpcodeLocation::Acir(1)],
+                count: 20,
+            },
+            Sample {
+                opcode: AcirOrBrilligOpcode::Acir(AcirOpcode::MemoryInit {
+                    block_id: BlockId(0),
+                    init: vec![],
+                    block_type: acir::circuit::opcodes::BlockType::Memory,
+                }),
+                call_stack: vec![OpcodeLocation::Acir(2)],
```

Comment:
Would be good to have a sample with the same callsite to see the count go up but not going to block with as I can see that it works from the posted flamegraph.